### ISSUE TITLE
Clients not syncing

### ISSUE THREAD

**Issue by eivindotcom** [mutability/mlat-client#25]
Hi. 
I have 4 clients i four different locations, aprox 40-60 km in straight line between them.  All the receivers running mlan-client and dump1090-mutability. On the server side mlat-server. All clients connect successfully to the server. When running ./view1090 on the clients I can see all the receivers receive the same airplanes.

On the server, in /sync/, it shows gray in the colums. Why dosent they sync and why no traffic to the server?
Here is a print from one client
`
1. Fri May 26 19:44:54 2017 Receiver status: connected 
2. Fri May 26 19:44:54 2017 Server status:  ready
3. Fri May 26 19:44:54 2017 Receiver:  16.0 msg/s received  5.5 msg/s processed (34%)
4. Fri May 26 19:44:54 2017 Server:  0.0 kB/s from server  0.0kB/s TCP to server  0.0kB/s UDP to server
5. Fri May 26 19:44:54 2017 Aircraft: 0 of 0 Mode S, 0 of 4 ADS-B used

One single time, i got " clock sync : 10 us between 2 of the receivers" on the server side.

**dump1090 settings** ./dump1090 --net --interactive
**clients settings** ./mlat-client --input-type dump1090 --input-connect 127.0.0.1:30005 --lat 5x.xxxx --lon 1x.xxxxx --alt 165 --server xx.xxx.xxx.xxx:yyyy --user xxx
**Server settings** ./mlat-server --client-listen xxxx:xxxx --work-dir /home/xxx/mlat-server/

**Comment by mutability** [comment]
That is a very low message rate.

"0 of 4 ADS-B used" implies that this client can see 4 ADS-B aircraft, but none of the other clients see those aircraft.

**Comment by eivindotcom** [comment]
Yes.. four clients ran for 15 minutes now, every client saw the same aircraft, and from view1090 , all the clients received over 500 msg ( some up to 2000msg ) from that aircraft... 

Is the settings I posted looking ok?

Update**
Receivers now syncing after opening UDP port on server netwok, not only TCP. Between 0.2 - 0.6 ( estimated synchronization error ) and 15 - 90 ppm. Added a 5 client, and it started acquring.
